script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_9_1.txt

```diff
@@ -51,11 +51,11 @@ As the pain grows even more unbearable, I take off my glasses.
 <Page10>
 The elevator comes into view.
 It's a long corridor.
-There are more than 20 meters remaining between us.
+More than 20 meters remain between me and the elevators.
```

Comment:
More than 20 meters still remain between me and the elevators.

Reply:
hrm, there's a `still` on the next line though which made it seem a bit redundant.